sections/package.zsh

```diff
@@ -100,9 +100,11 @@ spaceship_package::julia() {
 spaceship_package::maven() {
   spaceship::upsearch -s pom.xml || return
 
-  local maven_exe=$(spaceship::upsearch mvnw) || (spaceship::exists mvn && maven_exe="mvn") || return
-
-  $maven_exe help:evaluate -q -DforceStdout -D"expression=project.version" 2>/dev/null
+  local maven_exe=$(spaceship::upsearch mvnw || (spaceship::exists mvn && echo "mvn")) || return
```

Comment:
This is a very complex line. Can we simplify that with a multi-line `if-else`?

Reply:
Sorry @denysdovhan I totally missed this. Sure, I can clean things up. 👍

Reply:
Not a problem. Waiting for your changes.

Reply:
Sorry @denysdovhan but I had completely forgotten about this one. I have now simplified the expression to check if any of the mvn executable exists. Added some extra test cases to.